java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/Instance.java

```diff
@@ -544,16 +543,12 @@ public Class<?> apply(String typeId) {
         //
         // To convert a Slice type ID into a Java class, the following steps are taken:
         //
-        // 1. Convert the Slice type ID into a classname (e.g., ::M::X becomes M.X).
-        // 2. Check if the application has defined any package prefixes for the top-level module
-        // (e.g.:
-        // "M").
-        //    Attempt to resolve the <package-prefix>+<class-name> in order trying each defined
-        // package
-        //    prefix.
-        // 3. If the above step fails, it checks the value of Default.Package property. If found,
-        //    prepend its value to the classname. Otherwise, attempt to use the
-        //    classname directly.
```

Comment:
Or sometimes we have explicit lists of things, where again, the numbers really shouldn't be scattered inside the body of a monolithic paragraph.